src/main/resources/proto/googleidentity/oauth2/token/OAuth2Token.proto

```diff
@@ -0,0 +1,51 @@
+//   Copyright 2020 Google LLC
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto3";
+
+package googleidentity.oauth2.token;
+
+option java_package = "com.google.googleidentity.oauth2.token";
+option java_multiple_files = true;
+
+// Store oauth2 token information
+message OAuth2Token {
+
+    // OAuth2AccessToken value
+    string access_token = 1;
```

Comment:
can access_token and refresh_token be set at the same time or it's [oneof](https://developers.google.com/protocol-buffers/docs/overview#oneof) relationship?

Reply:
They can be set at the same time.

Reply:
But if we would keep multiple access tokens for one refresh_token, I may change it to oneof or maybe split refresh token and access token to two classes. Should I do that?

Reply:
As we chatted offline, yes you can split them